source/code-generation/query-converter/enable-query-converter.txt

```diff
@@ -0,0 +1,49 @@
+.. _rm-enable-query-converter:
+
+======================
+Enable Query Converter
+======================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+You can use the query converter to convert SQL statements, views, and 
+stored procedures to MongoDB syntax. Query converter considers the mapping 
+rules and schema transformations defined in your project when converting
+your SQL code.
```

Comment:
Can we update the wording here to be consistent with the introduction on the Convert Queries page?